pkg/webservice/handlers_test.go

```diff
@@ -2007,6 +2016,14 @@ func TestSpecificGroupResourceUsage(t *testing.T) {
 	assert.Equal(t, http.StatusBadRequest, resp.statusCode, statusCodeError)
 	assert.Equal(t, errInfo.Message, "invalid URL escape \"%Zt\"", jsonMessageError)
 	assert.Equal(t, errInfo.StatusCode, http.StatusBadRequest)
+
+	// Test invalid group name according to GroupRegExp
+	invalidGroupName := "invalid_group@name"
+	req, err = createRequest(t, "/ws/v1/partition/default/usage/group", map[string]string{"user": "testuser", "group": invalidGroupName})
+	assert.NilError(t, err)
+	resp = &MockResponseWriter{}
+	getGroupResourceUsage(resp, req)
+	assertInvalidGroupName(t, resp)
```

Comment:
I think this piece of code duplicates another [one](https://github.com/apache/yunikorn-core/pull/905/files#diff-c547735b58862d5573852f5e2c6a17c723f81a5620f3658172dd1b321f035975R2016-R2021).

I think the change of line [2001](https://github.com/apache/yunikorn-core/pull/917/files#diff-c547735b58862d5573852f5e2c6a17c723f81a5620f3658172dd1b321f035975R2001) is enough to cover what you implemented in [handlers.go](https://github.com/apache/yunikorn-core/pull/917/files#diff-c547735b58862d5573852f5e2c6a17c723f81a5620f3658172dd1b321f035975R2001)

Reply:
Many thanks for your suggestion. The test for InvalidGroupName is already exist. So I just deleted the new created one. Update the PR.